### PR TITLE
Mfranz1/logout fix

### PR DESCRIPTION
**Pull request by mfranz1**
Patch I
This patch attempts to enforce the deletion of the cookie upon logging out of CLARK.

Patch II: 
There is also a patch for processing body requests on the user profile

image
image
image

### CODE REVIEW

**Review by StarChaser44 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by EVogel1999 — COMMENTED** [review]
Just one question.

**Review by Cwagne17 — APPROVED** [review]

File: src/app/cube/user-profile/user-edit-information/user-edit-information.component.ts [hunk 1]
```diff
@@ -80,15 +78,21 @@ export class UserEditInformationComponent implements OnInit, OnChanges, OnDestro
...
+      username: this.user.username
     };
     try {
       await this.userService.editUserInfo(edits);
       await this.auth.validateAndRefreshToken();
       this.close.emit(true);
       this.noteService.success('Success!', 'We\'ve updated your user information!');
     } catch (e) {
-      this.noteService.error('Error!', 'We couldn\'t update your user information!');
+      console.log(e);
```
Comment (StarChaser44) [inline src/app/cube/user-profile/user-edit-information/user-edit-information.component.ts:90]: Unneeded console log?

**Review by StarChaser44 — COMMENTED** [review]
I had one question

File: src/app/core/auth.service.ts [hunk 2]
```diff
@@ -95,12 +95,14 @@ export class AuthService {
...
   private endSession() {
-    this.user = null;
+    this.user = undefined;
     this.openIdToken = null;
     this.cookies.remove('presence');
     this.changeStatus(false);
     this.group.next(AUTH_GROUP.VISITOR);
     localStorage.removeItem(TOKEN_STORAGE_KEY);
+    document.cookie =
+      'presence=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;';
```
[the remark this answers is not in the document]
Comment (mfranz1) [reply src/app/core/auth.service.ts:105]: Yes, this is the start of UTC time

**Review by EVogel1999 — APPROVED** [review]

**Pull request merged**